source/options.tsx

```diff
@@ -258,6 +258,29 @@ async function showStoredCssHotfixes(): Promise<void> {
 				: cachedCSS ?? 'No CSS found in cache.';
 }
 
+function enableToggleAll({currentTarget: button}: Event): void {
+	(button as HTMLButtonElement).parentElement!.remove();
+	for (const ui of select.all('.toggle-all-features')) {
+		ui.hidden = false;
+	}
+}
+
+function disableAllFeatures(): void {
+	for (const enabledFeature of select.all('.feature-checkbox:checked')) {
+		enabledFeature.click();
+	}
+
+	select('details#features')!.open = true;
+}
+
+function enableAllFeatures(): void {
+	for (const disabledFeature of select.all('.feature-checkbox:not(:checked)')) {
+		disabledFeature.click();
+	}
+
+	select('details#features')!.open = true;
```

Comment:
One day we'll use Preact and we'll stop eating spaghetti. 

That day is not today.